drivers/hmis/lib/form_data/default/fragments/disability.json

```diff
@@ -297,7 +297,23 @@
       ]
     },
     {
-      "fragment": "#hopwa_disability"
+      "type": "GROUP",
+      "link_id": "hopwa_section",
+      "text": "HOPWA Questions",
+      "enable_behavior": "ALL",
+      "enable_when": [
+        {
+          "question": "q_4_08_2",
+          "operator": "EQUAL",
+          "answer_code": "YES"
+        }
+      ],
+      "item": [
+        {
+          "fragment": "#hopwa_disability",
+          "_comment": "the hopwa_disability fragment also has a rule ensuring that it is only included for HOPWA-funded projects. For non-HOPWA programs, this whole hopwa_section group is excluded (because groups are excluded if all children are excluded)."
```

Comment:
oh, I forgot the rule should be coming from `hud_assessment_form_rules_2024.rb` because it is hud-specified, and we don't want users to be able to override it.

In that case I wonder whether the parallel rule applied in `hopwa_disability_for_annual.json` should also be put in that file?

Reply:
Yeah, I agree we shouldn't add the rule in two places. Maybe my suggestion is just to update the comment here, for example "the hopwa disability fragment also has a rule ensuring..." -> "the hopwa_disability fragment also has a rule (defined in code, not json) ensuring..." or something similar? But I also could be convinced, maybe what you have is fine and clear enough.

Reply:
Alright I updated the comments, I think it should be clearer now https://github.com/greenriver/hmis-warehouse/pull/4653/commits/b44cdcd3a92961d4c88d72a59c14f90cd5551b06